config_defaults_inc.php

```diff
@@ -3618,6 +3618,15 @@
  */
 $g_my_view_boxes_fixed_position = ON;
 
+/**
+ * Toggle whether 'Timeline' boxes show summary of issue after issue ID
+ */
+$g_timeline_show_issue_summary = OFF;
+
+/**
+ * Control the maximum length of each summary shown in timeline when $g_timeline_show_issue_summary is ON.
+ */
+$g_timeline_issue_summary_length = 20;
```

Comment:
At least the part of the CI process that builds the documentation did not show any errors
https://travis-ci.org/mantisbt/mantisbt/jobs/254667677

You would have to setup Publican to see the final documentation on your system.
Should be no big deal if you are running Linux, for more details see https://github.com/mantisbt/mantisbt/blob/master/docbook/README.md